### DISCUSSION TITLE
Bubble Mini (FW 3.2) isn't detecting Libre 2 in xdrip+

### DISCUSSION THREAD

**Discussion by aslepykh** [NightscoutFoundation/xDrip#3427] [Bug reports]
Hi!
I have an issue. My Bubble Mini (FW 3.2) isn't detecting Libre 2 in xdrip+ app. I'm using xdrip+ app on Xiaomi MI A3 (Android 11), it's without NFC, which was absolutely fine when I was using Libre 1 with Bubble Mini. All apps are up-to-date.
I tried OOP1 and OOP2 but still no positive results. 
Do you have any suggestions?
Thanks!

**Comment by aslepykh** [comment]
![Screenshot_20240409-191340](https://github.com/NightscoutFoundation/xDrip/assets/145323274/adf09a0c-9caf-4026-bcc6-9c3dcbeffe04)

**Reply by PhilTU** [reply]
Is this the first Libre 2 you have used ? Firmware 3.8 seems to be current at https://github.com/bubbledevteam/BubbleFW_FlashTool/releases but I don't use Bubble. OOP2 should be from May 2023. Show us the error log might help.

On the face of it "sensor not found" sounds like a hardware issue. Can you charge it, reset it and repostiion it ? Edit I just noticed it has read the serial number - is that correect for your sensor ?

Which part of the worls are you in ? Diabox might provide a second opinion about the hardware and sensor https://github.com/Bubblandevteam/diabox

**Reply by aslepykh** [reply]
Yeah, it's my first time using Libre 2. FW 3.8 is for Bubble, not Bubble Mini. Where do I find error logs in xdrip?

I fully charged Bubble Mini, tried resetting and repositioning and still nothing with Libre 2. Serial number is correct. 

I'm from Russia.

**Reply by PhilTU** [reply]
error event log is from menu top left three dots/burger menu.  Libre 2 serial number should start with 3 but xDrip misreports in some cases.

OOP2 version ?

**Reply by aslepykh** [reply]
Yeah, you are right about serial number, first number should be 3 but the rest is correct.
OOP2 version 27da3f5-2023.05.25

**Reply by PhilTU** [reply]
OOP2 is good then. Only leaves the log. 

Diabox lives at https://t.me/DiaboxRelease/18 for a second opinion.  Presumably sensor is started and reads ok in Librelink ?

**Reply by aslepykh** [reply]
Yeah, LibreLink 2 works just fine.
1st screenshot is after installing Libre 2.
2nd - Continous error messages "No sensor has been found".

![Screenshot_20240411-123547](https://github.com/NightscoutFoundation/xDrip/assets/145323274/5838fbda-14c2-4809-acd5-498f4db34b1e)
![Screenshot_20240411-123658](https://github.com/NightscoutFoundation/xDrip/assets/145323274/e0dfd31e-237f-4cec-b5e8-85f8d41c0ca3)

**Reply by aslepykh** [reply]
Diabox sees nothing too.

![Screenshot_20240411-124633](https://github.com/NightscoutFoundation/xDrip/assets/145323274/66160d90-74c8-4492-9b9e-25f72ebc4160)

**Reply by PhilTU** [reply]
ok, so the Bubble is saying "I can't see a sensor". I don't think this is a software issue. The Diabox developer posted this a couple of years ago when a number of similar reports were received - it's about positioning and gap. But not all cases were due to this. 
![179892584_574484980610251_2014597973125500746_n](https://github.com/NightscoutFoundation/xDrip/assets/76240903/9db23a2b-dc1c-4499-9407-6f4fd5809cc4)

**Reply by aslepykh** [reply]
Hmmm, I just can't get it) 
It was totally fine with Libre 1, no positioning issues. I'll try to reposition Libre 2

**Reply by PhilTU** [reply]
yes it is confusing. But the transmitter is telling the software "no sensor" so the software has nothing to work with.

Good luck. Apparently the gap marked in green above is important.

**Comment by Navid200** [comment]
@PhilTU
